README.md

```diff
@@ -138,22 +138,31 @@ This plugin provides 2 commands
 
 This is the command the mappings call under the hood. You can use it directly
 and prefix it with a count to target a specific terminal. This function also takes
-a the `size` and `dir` as an argument e.g.
+arguments `size`, `dir` and `direction`. e.g.
 
 ```vim
-:ToggleTerm size=40 dir=~/Desktop
+:ToggleTerm size=40 dir=~/Desktop direction=horizontal
 ```
 
+If `dir` is specified on creation toggle term will open at the specified directory.
+
 The directory can also be specified as `git_dir` which toggleterm will then
 use to try and derive the git repo directory.
 *NOTE*: This currently will not work for work tree's or other more complex setups
 
-If specified on creation toggle term will open at the specified directory at the
-specified height.
+If `size` is specified and the command opens a split (horizontal/vertical) terminal,
+the height/width of all terminals in the same direction will be changed to `size`.
+
+If `direction` is specified and the command opens a terminal,
+the terminal will be changed to the specified direction.
 
 _NOTE_: If the terminal has already been opened at a particular directory it will
 remain in that directory.
 
+_NOTE_: `size` and `direction` are ignored if the command closes a terminal.
+
+_NOTE_: Support for having terminals with different directions open at the same time is currently buggy.
```

Comment:
I think having 3 separate note sections is a bit strange. I think these can all be consolidated into on heading that just explains things to note, like `Caveats`. Also, I think the wording that something is buggy doesn't sound very good. I'd rather just say it's not supported currently.